patterns/vsi/config.tf

```diff
@@ -46,6 +46,19 @@ locals {
   }
   override_type = var.override_json_string == "" ? "override" : "override_json_string"
 
+
+  ##############################################################################
+  # Default SSH key
+  ##############################################################################
+
+  ssh_keys = [
```

Comment:
Was there a reason why you moved this block out of "local.config" block? For consistency I think it still belongs under the "local.config" as that is where all the other defaults are configured for the pattern.

Reply:
Yes Todd, If this was not moved, I was getting below error during plan phase itself: 
`Error: Local value local.config cannot use its own result as part of its expression.`